.github/workflows/test.yaml

```diff
@@ -0,0 +1,79 @@
+name: Test Examples
+
+on:
+  pull_request:
+  workflow_dispatch:
+
+jobs:
+  test-tcp-echo:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Create k8s Kind Cluster
+        uses: helm/kind-action@v1
+      - name: Setup
+        working-directory: tcp.echo
+        run: ./setup.sh
+      - name: Execute Test
+        working-directory: tcp.echo
+        run: |
+          set -o pipefail
+          ./test.sh | tee $GITHUB_STEP_SUMMARY
+      - name: Teardown
+        if: always()
+        working-directory: tcp.echo
+        run: ./teardown.sh
+
+  test-tls-echo:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Create k8s Kind Cluster
+        uses: helm/kind-action@v1
+      - name: Setup
+        working-directory: tls.echo
+        run: ./setup.sh
+      - name: Execute Test
+        working-directory: tls.echo
+        run: |
+          set -o pipefail
+          ./test.sh | tee $GITHUB_STEP_SUMMARY
+      - name: Teardown
+        if: always()
+        working-directory: tls.echo
+        run: ./teardown.sh
+
+  test-http-kafka-sync:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Install apt packages
+        uses: awalsh128/cache-apt-pkgs-action@latest
+        with:
+          packages: librdkafka-dev libyajl-dev
+          version: 1.0
+      - name: Install kcat
+        working-directory: /tmp
+        run: |
+          set -x
+          curl -L -o kcat https://github.com/attilakreiner/kcat/releases/download/1.7.1/kcat-linux-$(arch)
+          chmod +x kcat
+          sudo mv kcat /usr/local/bin
+          kcat 2>&1 | grep "Version 1.7.1"
```

Comment:
Can we use `apt` package `kafkacat` instead of this manual install?

https://github.com/edenhill/kcat?tab=readme-ov-file#install

Reply:
The apt package `kafkacat` install kafkacat version 1.6.0 that means it's lagging ca. one year behind the latest release. Even if we installed that we'd need to do some manual steps (e.g. renaming the executable from kafkacat to kcat) to keep the test in sync with the README and executable on other systems (e.g. on a Mac which has the latest kcat installed with brew).

Reply:
Actually I tested it, and that approach runs the test green, too. This is the alternative configuration:
```
      - name: Install apt packages
        uses: awalsh128/cache-apt-pkgs-action@latest
        with:
          packages: librdkafka-dev libyajl-dev kafkacat
          version: 1.0
      - name: Install kcat
        working-directory: /tmp
        run: |
          sudo ln -s /usr/bin/kafkacat /usr/local/bin/kcat
          kcat 2>&1 | grep "Version 1.6.0"
```
Pls LMK which one you prefer.

Reply:
I reverted the apt install commit as discussed, so going back to manual installation of kcat, as the decision was to install the latest version.